components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/PostAuthnMissingClaimHandler.java

```diff
@@ -444,7 +453,15 @@ protected void handlePostAuthenticationForMissingClaimsResponse(HttpServletReque
                 UserRealm realm = getUserRealm(user.getTenantDomain());
                 AbstractUserStoreManager userStoreManager = (AbstractUserStoreManager) realm.getUserStoreManager();
 
+                IdentityUtil.threadLocalProperties.get().put("isProgressiveProfileVerification", "true");
```

Comment:
Shall we move this to a constant `isProgressiveProfileVerification`?
And use the same constant in the governance PR?

Reply:
Addressed with commit 7e4de0f783087a70f53ed449163129e0845c271d